transports/Dockerfile

```
@@ -1,64 +1,47 @@
 # --- First Stage: Builder image ---
-FROM golang:1.24 AS builder
+FROM golang:1.24-alpine AS builder
 WORKDIR /app
 
+# Install dependencies in a single layer
+RUN apk add --no-cache git ca-certificates upx tzdata
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)